recipes/bintrees/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "bintrees" %}
+{% set version = "2.0.7" %}
+{% set sha256 = "60675e6602cef094abcd38bf4aecc067d78ae2d5e1645615c789724542d11270" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+  run:
+    - python
+
+test:
+  imports:
+    - bintrees
+    - bintrees.bintree
+    - bintrees.avltree
+    - bintrees.rbtree
+
+about:
+  home: https://pypi.python.org/pypi/bintrees/2.0.7
```

Comment:
Looks like the home page is (  https://github.com/mozman/bintrees.git ). Could you please update this in the feedstock?

Reply:
Raises as issue ( https://github.com/conda-forge/bintrees-feedstock/issues/1 ) on the feedstock.